ipc/tarpc/tarpc/src/server.rs

```diff
@@ -760,7 +760,9 @@ impl<Req, Res> InFlightRequest<Req, Res> {
         let method = serve.method(&message);
         // TODO(https://github.com/rust-lang/rust-clippy/issues/9111)
         // remove when clippy is fixed
+        #[allow(unknown_lints)]
         #[allow(clippy::needless_borrow)]
+        #[allow(clippy::needless_borrows_for_generic_args)]
         span.record("otel.name", &method.unwrap_or(""));
```

Comment:
Our current MSRV is 1.69, which doesn't know about `clippy::needless_borrows_for_generic_args`, so we also have to allow `unknown_lints`.